olm-catalog/serverless-operator/stopbundle.Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM alpine
```

Comment:
is that "OK" ?

Reply:
Why would it not be? It's only used in our CI and the only reason this is not `scratch` is us needing `sed`.

Reply:
Just asking, in case some "policies" ... :smile: 

thanks for the PR!

/lgtm